### PR TITLE
add optimization changes

### PR DESCRIPTION
**Pull request by projectoak2x**
- Auto Image Optimization
- Auto Generate Critical CSS
- Auto Sitemap Generator
- Auto Robots Generator
- CSS and JS Minify
- CSS Auto Prefixer
- Netlify Plugin Cache
- Local HTML Validation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
-This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
\ No newline at end of file
+  
+
+This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
+
+## Optimization Changes
+
+
+- [Auto Image Optimization](#auto-image-optimization)
```
Comment (gamrothc) [inline README.md:13]: can you add more to this ReadMe. For each of these the following two pieces of information would be helpful
1. why this is important
2. how to do it

File: README.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+- Auto Sitemap Generator
+- Auto Robots Generator
+- CSS and JS Minify
+- CSS Auto Prefixer
+- Netlify Plugin Cache
+- [Local HTML Validation](#local-html-validation)
+
+
+### Auto Image Optimization
+ `data-img2picture-ignore` - add this in the `img` tag for icons, like star, phone, pin or any small icons.
```
Comment (gamrothc) [inline README.md]: can you add more of a description to these. Also I think we should come up with a better name. If this is just for icons then we should have icons in the name

File: src/config/plugins/image.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+/**
+ *  A NOTE ON THE IMAGE PLUGIN
+ *  https://github.com/saneef/eleventy-plugin-img2picture
+ *
+ *      The image optimization plugin allows you to automatically generate responsive, optimised images
+ *      for your website. It works by taking a standard <img> tag, optimising the images, then replacing
+ *      the <img> with a <picture> and <source>'s.
+ *
+ *      Due to the extra configuration needed to get this to work, we've set this up as an opt-in feature.
```
Comment (gamrothc) [inline src/config/plugins/image.js]: when you say "opt-in feature" what does this mean? How does someone opt-in

File: README.md (hunk 1 above)
Comment (bruce070124) [reply README.md:13]: Hello Collin @gamrothc  Thanks for reviewing this. I've added and updated the readme , also updated other optimization why they are added and how it works. Some doesn't have "how to do" it as it doesn't require any additional configuration for the dev. Thanks

File: README.md (hunk 2 above)
Comment (bruce070124) [reply README.md]: Add additional info for this btw, also we can't really change the `data-img2picture-ignore` name as it's based on the package we use https://github.com/saneef/eleventy-plugin-img2picture

File: src/config/plugins/image.js (hunk 3 above)
Comment (bruce070124) [reply src/config/plugins/image.js]: My bad I just remove this text on the comment btw as it's not an opt-in feature anymore. We have enabled it by default for this update.

opt-in feature means optional.

**Pull request merged**